src/pages/Accounts/Fixed/Fixed.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { requestForFixedAccount } from '../../../redux/actions/Fixed';
```

Comment:
Add this export requestForFixedAccount in the index file inside actions and then import from the index.

Reply:
changes updated.